releasenotes/notes/update-pipe-name-env-c446344396e8567a.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Rename experimental configuration `dogstatsd_windows_pipe_name` to `dogstatsd_pipe_name`.
```

Comment:
Since `dogstatsd_windows_pipe_name` was not documented on the datadog.yaml template nor in any releasenotes do we need to add an upgrade releasenote ? Was it documented elsewhere ?

Reply:
changelog removed.